addons/web/tests/test_db_manager.py

```diff
@@ -0,0 +1,117 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+import requests
+
+import odoo
+
+from odoo.tests.common import BaseCase, HttpCase, tagged
+from odoo.tools import config
+
+from unittest.mock import patch
+
+
+class TestDatabaseManager(HttpCase):
+    def test_databasdbe_manager(self):
+        res = self.url_open('/web/database/manager')
+        self.assertEqual(res.status_code, 200)
+        page = str(res.content)
+
+        # check that basic existing db actions are present
+        self.assertIn('.o_database_backup', page)
+        self.assertIn('.o_database_duplicate', page)
+        self.assertIn('.o_database_delete', page)
+
+        # check that basic db actions are present
+        self.assertIn('.o_database_create', page)
+        self.assertIn('.o_database_restore', page)
+
+
+@tagged('-at_install', 'post_install', '-standard', 'database_operations')
+class TestDatabaseOperations(BaseCase):
+    def setUp(self):
+        self.password = 'test_master_passwd'
```

Comment:
Could we randomize this with e.g. `secrets.token_hex()`, please? It should work equally well.
I know, I know, it's just tests, but it doesn't hurt to do it, right? ;)

Reply:
Better safe than sorry